ern-api-gen/src/DefaultGenerator.ts

```diff
@@ -318,9 +318,12 @@ export default class DefaultGenerator extends AbstractGenerator {
           } catch (e) {
             rethrow(
               e,
-              "Could not process model '" +
-                name +
-                "'.Please make sure that your schema is correct!",
+              [
+                `Could not process model '${name}'`,
+                'Please make sure that your schema is correct!',
+                `Failed with error message: ${(e && e.message) ||
+                  'unknown error'}`,
+              ].join('\n'),
```

Comment:
this has been _super helpful_ in assisting to see _why_ a generation error occurred!

Reply:
@friederbluemle--it's not really _upstream_ though ya?  not sure who to describe this.  is it a mimicked implementation?  is there some sort of java -> ts synchronization process?

Reply:
incredible, i was unaware of that setting.  will be turning that on!  this could be scrapped then if it makes it into the trace output

Reply:
i was getting ready to send another PR to restore this simple error message, but it came upon me that the error output here at a non-trace level does infact belong.  the point of this message here is for user UX, not library dev UX.  as it stands, the user UX requires the error message up front, and we hide it from them. i'll see about a patch to swagger proper